gravitee-apim-portal-webui-next/src/app/catalog/catalog.component.ts

```diff
@@ -53,6 +63,7 @@ export class CatalogComponent {
   // TODO: Get banner title + subtitle from configuration
   bannerTitle: string = 'Welcome to Gravitee Developer Portal!';
   bannerSubtitle: string = 'Discover powerful APIs to supercharge your projects.';
+  selectedFilter: string = 'all';
```

Comment:
Just a heads up, not a necessary fix at all: this could be transformed to a signal and in the constructor, and `effect( () => {} )` added to the constructor to handle changes.